src/test/java/com/google/api/codegen/testsrc/common/samples/delete_shelf.sample.yaml

```diff
@@ -0,0 +1,51 @@
+type: com.google.api.codegen.samplegen.SampleConfigProto
+config_schema_version: 1.2.0
+samples:
+  # Behavior change:
+  # gapic config: if region_tag is unspecified or region_tag is not unique, derive the 
```

Comment:
There is some trailing whitespace in some lines in some of these files. If you could remove it, that would be great.